suite-native/atoms/src/DiscreetText.tsx

```diff
@@ -121,7 +128,7 @@ export const DiscreetText = ({
                 variant={variant}
                 color={color}
                 onLayout={handleLayout}
-                style={applyStyle(textStyle, { isDiscreetMode })}
+                style={applyStyle(textTemplateStyle, { isDiscreetMode })}
```

Comment:
Variable isDiscreetMode seems unused inside of style

Reply:
fixed in 94ef45ffb